mappings/net/minecraft/client/option/Option.mapping

```diff
@@ -58,6 +58,8 @@ CLASS net/minecraft/class_316 net/minecraft/client/option/Option
 	FIELD field_26925 HIDE_MATCHED_NAMES_TOOLTIP Lnet/minecraft/class_2561;
 	FIELD field_27956 TOGGLE_TEXT Lnet/minecraft/class_2561;
 	FIELD field_27957 HOLD_TEXT Lnet/minecraft/class_2561;
+	FIELD field_32146 MONOCHROME_LOGO Lnet/minecraft/class_4064;
+	FIELD field_32148 MONOCHROME_LOGO_TOOLTIP Lnet/minecraft/class_2561;
```

Comment:
Wouldn't `MONOCHROME_BACKGROUND` be better here? I think it is called Monochrome Background, not logo in-game.

Reply:
It is called 'Monochrome Logo' in the Accessibility Settings screen:

![Monochrome Logo option](https://user-images.githubusercontent.com/24855774/113608225-47a04780-9618-11eb-891d-cf1d3cd68e29.png)

Reply:
No, actually it's called Monochrome Logo